changelog.d/10566.feature

```diff
@@ -0,0 +1 @@
+Allow room creators to send historical events specified by [MSC2716](https://github.com/matrix-org/matrix-doc/pull/2716) in existing room versions.
```

Comment:
@erikjohnston I need to add some Complement tests for this to make sure this actually works but how does the code look?

Reply:
Tests added in https://github.com/matrix-org/complement/pull/192 😀